### ISSUE TITLE
Has anyone tried modifying the hyper-parameters of the model architecture.

### ISSUE THREAD

**Issue by programindz** [k2-fsa/icefall#1987]
I'm trying but the grad scale is becoming smaller and smaller even when i decrease the learning rate. I'm primarily targeting to reduce the heads and embedding dimensions.

**Comment by csukuangfj** [comment]
what is your --max-duration?

same issue as https://github.com/k2-fsa/icefall/issues/1993
